tests/optimize/test_hyperopt.py

```diff
@@ -189,6 +191,8 @@ def test_start_no_data(mocker, hyperopt_conf) -> None:
     with pytest.raises(OperationalException, match='No data found. Terminating.'):
         start_hyperopt(pargs)
 
+    os.unlink(Hyperopt.get_lock_filename(hyperopt_conf))
```

Comment:
How's the changes in this file related to this PR? 

if it's not (and just fixing some random test failure) - please create a separate PR for that ...

Reply:
Yes, my bad.